README.md

```diff
@@ -52,6 +52,54 @@ Documentation and code samples for these libraries can be found [here](https://a
 * File an issue via [Github Issues](https://github.com/Azure/azure-sdk-for-net/issues/new/choose).
 * Check [previous questions](https://stackoverflow.com/questions/tagged/azure+.net) or ask new ones on StackOverflow using `azure` and `.net` tags.
 
+
+## Data Collection
+The software may collect information about you and your use of the software and send it to Microsoft. Microsoft may use this information to provide services and improve our products and services. You may turn off the telemetry as described below. You can learn more about data collection and use in the help documentation and Microsoft’s [privacy statement](https://go.microsoft.com/fwlink/?LinkID=824704). For more information on the data collected by the Azure SDK, please visit the [Telemetry Guidelines](https://azure.github.io/azure-sdk/general_azurecore.html#telemetry-policy) page.
+
+### Telemetry Configuration
+Telemetry collection is on by default.
+
+To opt out, you can disable telemetry at client construction. Create a `ClientOptions` for your specific client, every service client has an option class. Set `ApplicationId` as empty under `Diagnostics` inside service client option. Then pass service client option as argument during client creation. This will disable telemetry for all methods in the client. Do this for every new client.
```

Comment:
This is not universally accurate.   For example, it does not apply to the AMQP-based clients.